config/gasSchedule_test.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/stretchr/testify/assert"
+)
+
+type operations struct {
+	OperationA uint64
+	OperationB uint64
+	OperationC uint64
+	OperationD uint64
+	OperationE uint64
+}
+
+func TestDecode(t *testing.T) {
+	gasMap := make(map[string]uint64)
+	gasMap["OperationB"] = 4
+	gasMap["OperationA"] = 3
+	gasMap["OperationC"] = 100
+	gasMap["OperationD"] = 1000
+
+	op := &operations{}
+	err := mapstructure.Decode(gasMap, op)
+	assert.Nil(t, err)
+
+	fmt.Printf("%+v\n", op)
+}
+
+func TestDecode_ArwenGas(t *testing.T) {
+	gasMap := make(map[string]uint64)
+	gasMap["StorePerByte"] = 4
+	gasMap["GetOwner"] = 4
+	gasMap["GetExternalBalance"] = 4
+	gasMap["BigIntByteLength"] = 4
+
+	bigIntOp := &BigIntAPICost{}
+	err := mapstructure.Decode(gasMap, bigIntOp)
+	assert.Nil(t, err)
+
+	fmt.Printf("%+v\n", bigIntOp)
+
+	erdOp := &ElrondAPICost{}
+	err = mapstructure.Decode(gasMap, erdOp)
+	assert.Nil(t, err)
+
+	fmt.Printf("%+v\n", erdOp)
+
+	ethOp := &EthAPICost{}
+	err = mapstructure.Decode(gasMap, ethOp)
+	assert.Nil(t, err)
+
+	fmt.Printf("%+v\n", ethOp)
+}
```

Comment:
Optionally add a test with a `gasMap` that contains a `0` field and assert that  `CreateGasConfig()` returns an error.